src/axom/sina/core/CurveSet.cpp

```diff
@@ -57,41 +61,64 @@ void addCurve(Curve &&curve, CurveSet::CurveMap &curves)
  *
  * @param parent the parent node
  * @param childNodeName the name of the child node
- * @return a CurveMap representing the specified child
+ * @return a struct containing the curveMap and curveOrder.
```

Comment:
On review, there's some confusing use of `curveOrder` here and elsewhere as both the enum and the vector of ordered curve names--will go with `curveOrder` and `orderedCurveNames` in the next commit (or let me know if you have a preferred one)